src/libs/relay/c/conduit_relay_io.h

```diff
@@ -101,13 +101,13 @@ void CONDUIT_RELAY_API conduit_relay_io_load_step_and_domain(const char *path,
                                                              conduit_node *cnode);
 
 ///
-/// ``query_number_of_domains`` return the number of time steps.
+/// ``query_number_of_time_steps`` returns the number of time steps.
```

Comment:
Shouldn't this just say `query_number_of_steps`?

Reply:
Yes.